src/utils/gFormsAnswers.js

```diff
@@ -0,0 +1,52 @@
+const { google } = require("googleapis");
+
+const service = google.sheets("v4");
+const credentials = require("../config/CF_credentials.json");
+
+const authClient = new google.auth.JWT(
+    credentials.client_email,
+    null,
+    credentials.private_key.replace(/\\n/g, "\n"),
+    ["https://www.googleapis.com/auth/spreadsheets"]
+);
+
+async function authorize() {
+    const token = await authClient.authorize();
+    authClient.setCredentials(token);
+}
+
+authorize();
```

Comment:
would this not be better suited to be invoked with in the deps obj when you initialize all the other functions?